### ISSUE TITLE
How can i use it with fully convolutional network??

### ISSUE THREAD

**Issue by rameezrehman83** [jacobgil/keras-grad-cam#29]
I want to output grad-cam of U-net architecture which does't has sigmoid layer at the end and no fully connected and softmax layer.
